sites/hashdev/src/_pages/blog/5_build-blocks-in-any-language.mdx

```diff
@@ -3,7 +3,7 @@ authorPhoto: blog/authors/ahmad-sattar.png
 postPhoto: blog/5_build-blocks-in-any-language/FSharp_Cover.jpg
 title: "How to build Block Protocol blocks in any language"
 subtitle: "Transpiling from F# to JavaScript to build a complex spreadsheet block"
-author: "Ahmad Sattar Atta"
+author: "Senior Ahmad Sattar Atta"
```

Comment:
I think the "Senior" is placed in the wrong like.

Reply:
D'oh. Señor, maybe. Thanks for spotting this!

Reply:
```suggestion
author: "Ahmad Sattar Atta"
```